### ISSUE TITLE
cc_proto_blacklist_test.bzl doesn't exist

### ISSUE THREAD

**Issue by ehargitt** [protocolbuffers/protobuf#7179]
**What version of protobuf and what language are you using?**
Version: protobuf-all-3.11.3.tar.gz

**What operating system (Linux, Windows, ...) and version?**
Debian Stretch

**What runtime / compiler are you using (e.g., python version or gcc version)**
N/A

**What did you do?**
1. Download tarball from Github page
2. Untar tarball and enter directory
3. Do `bazel build //...`

**What did you expect to see**
Not errors.
**What did you see instead?**
Errors.
Make sure you include information that can help us debug (full error message, exception listing, stack trace, logs).
```bash
root@52f237e770c3:/app/protobuf-3.11.3# bazel build //...
Starting local Bazel server and connecting to it...
INFO: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /root/.cache/bazel/_bazel_root/51fd306b3bc99c94df428868d89a5855/external/bazel_tools/tools/build_defs/repo/http.bzl:292:16):
 - /app/protobuf-3.11.3/protobuf_deps.bzl:60:9
 - /app/protobuf-3.11.3/WORKSPACE:17:1
INFO: Call stack for the definition of repository 'rules_proto' which is a http_archive (rule definition at /root/.cache/bazel/_bazel_root/51fd306b3bc99c94df428868d89a5855/external/bazel_tools/tools/build_defs/repo/http.bzl:292:16):
 - /app/protobuf-3.11.3/protobuf_deps.bzl:52:9
 - /app/protobuf-3.11.3/WORKSPACE:17:1
INFO: Call stack for the definition of repository 'rules_cc' which is a http_archive (rule definition at /root/.cache/bazel/_bazel_root/51fd306b3bc99c94df428868d89a5855/external/bazel_tools/tools/build_defs/repo/http.bzl:292:16):
 - /app/protobuf-3.11.3/protobuf_deps.bzl:36:9
 - /app/protobuf-3.11.3/WORKSPACE:17:1
ERROR: error loading package '': Unable to load file '//:cc_proto_blacklist_test.bzl': file doesn't exist
INFO: Elapsed time: 5.606s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
    currently loading: util/python
```

**Anything else we should know about your project / environment**

**Comment by ehargitt** [comment]
Downloading `cc_proto_blacklist_test.bzl` from master seems to fix it (fail in a different way 😁)

**Comment by matshch** [comment]
Looks like same issue affects protobuf-all-3.11.4.tar.gz.

**Comment by matshch** [comment]
As I see, this bug was introduced in 29cd005ce1fe1a8fabf11e325cb13006a6646d59 (v3.11.3), and now it is fixed in f0cb9cdb957e27c191d67a3154b72f383a4ad60f (v3.12.0-rc1).
